cron/fetch_queries_from_elk

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/php
+<?php
+/**
+ * Retrieve user queries from openSearch logs stored in elk for a given date
+ */
+
+// currently 10000 is the max loop size, elastic search (elk) fails for larger chunks
+define('LOOP_SIZE', 10000);
+
+$opt = options();
+
+$extract_date = empty($opt['d']) ? strtotime("-1 days") : strtotime($opt['d']);
+$elk = $opt['e'] . '/k8s-cisterne-%s/_search?sort=@timestamp:asc';
+$pwd = $opt['p'];
+$out = fopen($opt['o'], 'w');
+if (!$out) die('ERROR. Cannot open output file: ' . $opt['o'] . "\n");
+
+$curl = set_curl($elk, $pwd, gmdate('Y.m.d', $extract_date));
+
+$sort = [];
+$lines = 0;
+do {
+  $extract = set_elk_parms($extract_date, $sort);
+  curl_setopt($curl, CURLOPT_POSTFIELDS, $extract);
+  $data = curl_exec($curl);
+  $c_info = curl_getinfo($curl);
+  if ($c_info['http_code'] != 200) die('http error ' . $c_info['http_code'] . ' fetching ' . $elk . "\n");
+  if (empty($data)) die('no data returned from ' . $elk . " even though http status was 200\n");
+  $json = json_decode($data);
+  $sort = [];
+  foreach ($json->hits->hits as $hit) {
+    $sort = $hit->sort;
```

Comment:
Hvad er meningen med at overskrive sort for hver `foreach` uden at gøre brug af den sort?